src/agentscope/models/dashscope_model.py

```diff
@@ -124,6 +125,8 @@ def __init__(
         self,
         config_name: str,
         model_name: str = None,
+        app_id: str = None,
+        api_type: Literal["generation", "application"] = "generation",
```

Comment:
Update the docstrings so that developers can know how to use this class as a model or application wrapper.